wgpu-hal/src/vulkan/adapter.rs

```diff
@@ -751,8 +871,26 @@ impl super::Instance {
             },
         };
 
+        let update_after_bind = if phd_capabilities.properties.vendor_id == db::intel::VENDOR {
```

Comment:
do we still need to limit this to Intel? If we are automatically enabling it only for limits exceeds, we should just do it for all vendors